ocpmodels/modules/evaluator.py

```diff
@@ -30,6 +30,12 @@
 
 
 class Evaluator:
+    atomic_number_map = {1: "H", 6: "C", 7: "N", 8: "O"}
```

Comment:
It might be best to move this to a data file.

Reply:
Alternatively, it would be good to make this configurable so we can select them at runtime.

Reply:
Got it.